### ISSUE TITLE
Failing to build with qmake

### ISSUE THREAD

**Issue by rannick1982** [qbasicer/qzdl#47]
/home/rannick1982/Downloads/qzdl/Makefile:701: warning: overriding recipe for target 'obj/qzdl.o'
/home/rannick1982/Downloads/qzdl/Makefile:663: warning: ignoring old recipe for target 'obj/qzdl.o'
:-1: error: No rule to make target '/src/qzdl.cpp', needed by 'obj/qzdl.o'.  Stop.

I have run into these issues when trying to build in terminal and in QT Creator, any suggestions on how to resolve these errors?

**Comment by finnoleary** [comment]
This problem still exists.

**Comment by SteelPh0enix** [comment]
Still having this problem, Ubuntu 18.04, with Qt toolchain created. ./configure tells me everything is OK, and then when i use `make`, this error shows up.

**Comment by SteelPh0enix** [comment]
All right, i did some stuff and this it what i've got

I'm doing everything under Ubuntu 18.04, with GCC 7.3.0 and Qt 5.9.5

I've used `./configure` to generate `*.pro` file, opened it with QtCreator.
I've got the same error as OP, so i've cleaned up the project (deleted Makefile and cleaned it), got another error about not finding some source files and headers. Looked up the `*.pro` file and seen that some paths have `/` at the beginning, which made them invalid, so i deleted it, and cleaned up  project again with running qmake. After that, i've got `QApplication: No such file or directory`, so i've moved `QT += [...]` from `*.pro` file at the top and added `core gui` to it, since there was only `network` module. After that, Qt still can't find QApplication, and i have no idea why.
